.changeset/tasty-jobs-help.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-graphql-voyager': minor
+---
+
+Add the new GraphQL Voyager tool as a plugin for backstage, expose necessary API and components needed for it.
```

Comment:
Can we include a comment around going to the `README.md` for installation instructions? This changeset will be used as release notes so for those upgrading this will be one way they learn about this new plugin so pointing them somewhere with more details is really helpful